contrib/vagrant/start.sh

```diff
@@ -280,7 +281,7 @@ function write_cilium_cfg() {
     ipv6_addr="${3}"
     filename="${4}"
 
-    cilium_options="--auto-ipv6-node-routes --debug-verbose flow"
+    cilium_options="--auto-ipv6-node-routes --debug-verbose flow --prometheus-serve-addr=:9090"
```

Comment:
Don't you need to to the same for the cilium-health options?

Reply:
I think I will just remove that option. Maybe we should consider adding some env variable (i.e. `PROMETHEUS`) for enabling Prometheus metrics in Vagrant environment, but not in this PR.